### ISSUE TITLE
Experiments on the Mouse Dataset

### ISSUE THREAD

**Issue by 15831959673** [lhy0322/DeepSEQ#1]
Hello, respected author. Because I rarely use the TensorFlow framework, I replicated your model using Pytorch. The performance of your model has been achieved on the human dataset, but it is far from achieving an accuracy of 84.6 on the mouse dataset.I also used many other model tests afterwards and found that the performance of human datasets can be achieved. But the mouse dataset will never surpass an accuracy of around 77. Moreover, the DeepSE model I implemented cannot achieve the accuracy of 82 mentioned in your paper. So I would like to ask you, can your model run stably with an accuracy of over 80
